src/ocm/components/hosts/ResetHostModal.tsx

```diff
@@ -28,7 +28,7 @@ const ResetHostModal: React.FC<ResetHostModalProps> = ({ isOpen, hostname, onClo
       </Button>,
     ]}
   >
-    Are you sure you want to reset host{` ${hostname}` || ''} ?
+    Are you sure you want to reset host{` ${hostname || ''}`} ?
```

Comment:
same as in https://github.com/openshift-assisted/assisted-ui-lib/pull/1456/files#r902576434